[2023-12-10 19:43:18,439] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-08T00:00:00+00:00 [queued]>
[2023-12-10 19:43:18,575] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-08T00:00:00+00:00 [queued]>
[2023-12-10 19:43:18,618] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:43:18,691] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:43:18,739] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:43:19,781] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-06-08 00:00:00+00:00
[2023-12-10 19:43:19,810] {standard_task_runner.py:52} INFO - Started process 3344 to run task
[2023-12-10 19:43:19,868] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-06-08T00:00:00+00:00', '--job-id', '940', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpz7fdbb3h', '--error-file', '/tmp/tmpnax7j0xi']
[2023-12-10 19:43:19,877] {standard_task_runner.py:80} INFO - Job 940: Subtask segunda_tarea
[2023-12-10 19:43:21,165] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-08T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:43:22,734] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-06-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-08T00:00:00+00:00
[2023-12-10 19:43:22,736] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:43:22,741] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:43:22,804] {subprocess.py:85} INFO - Output:
[2023-12-10 19:43:22,808] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:43:22,809] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:43:23,349] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230608T000000, start_date=20231210T194318, end_date=20231210T194323
[2023-12-10 19:43:23,564] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:43:24,568] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
